app/src/main/java/com/vultisig/wallet/ui/screens/ChainCoinScreen.kt

```diff
@@ -67,6 +72,9 @@ fun ChainCoinScreen(navController: NavHostController) {
     val textColor = MaterialTheme.colorScheme.onBackground
     val appColor = Theme.colors
     val dimens = MaterialTheme.dimens
+    val buyVltiButtonVisible = false
+    val canDeposit by viewModel.canDeposit.collectAsState()
+    val canSwap by viewModel.canSwap.collectAsState()
```

Comment:
canDeposit and canSwap can be a part of ChainCoinUiModel, so we can fully represent a screen state through its model.